MAUI/Maps/zoom-pan.md

```diff
@@ -0,0 +1,633 @@
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png">
+            <map:MapTileLayer.ZoomPanBehavior>
+                <map:MapZoomPanBehavior ZoomLevel="2" EnableDoubleTapZooming="True"/>
+            </map:MapTileLayer.ZoomPanBehavior>
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+MapZoomPanBehavior zoomPanBehavior = new MapZoomPanBehavior();
+zoomPanBehavior.ZoomLevel = 2;
+zoomPanBehavior.EnableDoubleTapZooming = true;
+tileLayer.ZoomPanBehavior = zoomPanBehavior;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Double tap zooming](images/zoom-pan/double_tap_zoom.gif)
+
+## Animation
+
+### Zoom level Animation
+
+To perform animation on zoom level changing using the `MapTileLayer.EnableZoomingAnimation` property. The default value of the `EnableZoomingAnimation` is `true`.
+
+N> It is applicable for both tile layer and shape layer.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+                          EnableZoomingAnimation="True" >
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+tileLayer.EnableZoomingAnimation = true;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Zoom animation](images/zoom-pan/zoom_animation.gif)
+
+### Center Animation
+
+To perform animation on center value changing using the `MapTileLayer.EnableCenterAnimation` property. The default value of the `EnableCenterAnimation` is `true`.
+
+N> It is applicable only for tile layer.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapTileLayer UrlTemplate="https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+                          EnableCenterAnimation="True" >
+        </map:MapTileLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfMaps map = new SfMaps();
+MapTileLayer tileLayer = new MapTileLayer();
+tileLayer.UrlTemplate = "https://tile.openstreetmap.org/{z}/{x}/{y}.png";
+tileLayer.EnableCenterAnimation = true;
+map.Layer = tileLayer;
+this.Content = map;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Center animation](images/zoom-pan/center_zoom_animation.gif)
+
+## Event
+
+The `ZoomLevelChanging` event triggers when zoom level changed. Following arguments can be get from the ZoomLevelChanging event .
```

Comment:
The `ZoomLevelChanging` event triggers when the zoom level is changed. Following arguments can be get from the ZoomLevelChanging event.

Reply:
Changed the content

Reply:
Extra space before `.`

Reply:
Removed